ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/legaladvisor/CCDOrderGenerationData.java

```diff
@@ -1,26 +1,47 @@
 package uk.gov.hmcts.cmc.ccd.domain.legaladvisor;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
 import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
 
 import java.time.LocalDate;
+import java.util.Collections;
 import java.util.List;
 
 @Data
+@JsonIgnoreProperties(ignoreUnknown = true)
 @Builder
+@AllArgsConstructor //see https://github.com/rzwitserloot/lombok/issues/1347
+@NoArgsConstructor
 public class CCDOrderGenerationData {
-
     private LocalDate docUploadDeadline;
+
     private CCDDirectionPartyType docUploadForParty;
+
     private LocalDate eyewitnessUploadDeadline;
+
     private CCDDirectionPartyType eyewitnessUploadForParty;
-    private List<CCDOrderDirectionType> directionList;
-    private List<CCDOrderDirection> otherDirectionList;
+
+    @Builder.Default
+    private List<CCDOrderDirectionType> directionList = Collections.emptyList();
```

Comment:
Don't we need `List<CCDCollectionElement< CCDOrderDirectionType >> directionList ` ?

Reply:
yes we need

Reply:
It seems like this is coming as a plain list of enums, so it's not a ccd collection element. `ccd /shrug`

Reply:
It seems like this is coming as a plain list of enums, so it's not a ccd collection element. ccd /shrug

Reply:
👍